### ISSUE TITLE
Port all themes in current website, to v2 website

### ISSUE THREAD

**Issue by pradyunsg** [sphinx-themes/sphinx-themes.org#25]
This issue contains the list of themes in the current v1 website. As we add themes to v2 (or decide to not add them due to issues), we'll document it here.

- [x] [agoraplex.themes.sphinx](https://pypi.org/p/agoraplex.themes.sphinx) -- cannot be installed anymore
- [x] [allanc-sphinx](https://pypi.org/p/allanc-sphinx) -- instructions in README are not sufficient
- [x] [astropy-sphinx-theme](https://pypi.org/p/astropy-sphinx-theme) -- #26
- [x] [cakephp_theme](https://pypi.org/p/cakephp_theme) -- https://github.com/cakephp/doc-theme/ no longer exists and instructions in README are not sufficient
- [x] [caktus-sphinx-theme](https://pypi.org/p/caktus-sphinx-theme) -- #26
- [x] [cloud_sptheme](https://pypi.org/p/cloud_sptheme) -- #53
- [x] [crate-docs-theme](https://pypi.org/p/crate-docs-theme) -- "Note: This theme is tightly integrated into the Crate.io website and is not intended for general use."
- [x] [edx-sphinx-theme](https://pypi.org/p/edx-sphinx-theme) -- #19
- [x] [epfl-sphinx-theme](https://pypi.org/p/epfl-sphinx-theme) -- no instructions on how to use
- [x] [f5-sphinx-theme](https://pypi.org/p/f5-sphinx-theme) -- "This theme should not be used without modification for any project that doesn't publish documentation to clouddocs.f5.com."
- [x] [flask-sphinx-themes](https://pypi.org/p/flask-sphinx-themes) -- #26
- [x] [foundation-sphinx-theme](https://pypi.org/p/foundation-sphinx-theme) -- causes ImportError when following instructions from README
- [x] [groundwork-sphinx-theme](https://pypi.org/p/groundwork-sphinx-theme) -- #26
- [x] [guzzle_sphinx_theme](https://pypi.org/p/guzzle_sphinx_theme) -- #26
- [x] [hachibee-sphinx-theme](https://pypi.org/p/hachibee-sphinx-theme) -- unclear instructions in README
- [x] [hbp-sphinx-theme](https://pypi.org/p/hbp-sphinx-theme) -- #35
- [x] [insegel](https://pypi.org/p/insegel) -- #19
- [x] [jupyter-sphinx-theme](https://pypi.org/p/jupyter-sphinx-theme) -- #36
- [x] [karma_sphinx_theme](https://pypi.org/p/karma_sphinx_theme) -- #37
- [x] [kotti_docs_theme](https://pypi.org/p/kotti_docs_theme) -- #38
- [x] [logilab-sphinx-themes](https://pypi.org/p/logilab-sphinx-themes) -- #19
- [x] [lsst-dd-rtd-theme](https://pypi.org/p/lsst-dd-rtd-theme) -- #19
- [x] [lsst-sphinx-bootstrap-theme](https://pypi.org/p/lsst-sphinx-bootstrap-theme)
- [x] ~[mkdocs-nature](https://pypi.org/p/mkdocs-nature)~
- [x] [mozilla-sphinx-theme](https://pypi.org/p/mozilla-sphinx-theme) -- #19
- [x] [msmb_theme](https://pypi.org/p/msmb_theme) -- no released version declares dependencies correctly.
- [x] [murray](https://pypi.org/p/murray) -- #41
- [x] [oe-sphinx-theme](https://pypi.org/p/oe-sphinx-theme) -- #42
- [x] [ovs-sphinx-theme](https://pypi.org/p/ovs-sphinx-theme) -- #43
- [x] [pietroalbini-sphinx-themes](https://pypi.org/p/pietroalbini-sphinx-themes) -- #44
- [ ] [psphinxtheme](https://pypi.org/p/psphinxtheme)
- [x] [pylons-sphinx-themes](https://pypi.org/p/pylons-sphinx-themes)
- [x] [python-docs-theme](https://pypi.org/p/python-docs-theme) -- #19
- [x] [quark-sphinx-theme](https://pypi.org/p/quark-sphinx-theme) -- #45
- [x] [renga-sphinx-theme](https://pypi.org/p/renga-sphinx-theme) -- #47
- [x] [renku-sphinx-theme](https://pypi.org/p/renku-sphinx-theme) -- renamed
- [x] [romnnn_sphinx_press_theme](https://pypi.org/p/romnnn_sphinx_press_theme)
- [x] [rtcat-sphinx-theme](https://pypi.org/p/rtcat-sphinx-theme)
- [x] [solar-theme](https://pypi.org/p/solar-theme) -- #48
- [x] [sphinx_adc_theme](https://pypi.org/p/sphinx_adc_theme) -- #49
- [x] [sphinx_bernard_theme](https://pypi.org/p/sphinx_bernard_theme) -- #19
- [x] [sphinx_celery](https://pypi.org/p/sphinx_celery) -- #19
- [x] [sphinx_drove_theme](https://pypi.org/p/sphinx_drove_theme) -- #50
- [x] [sphinx_hand_theme](https://pypi.org/p/sphinx_hand_theme)
- [x] [sphinx_materialdesign_theme](https://pypi.org/p/sphinx_materialdesign_theme) -- #51
- [ ] [sphinx_minoo_theme](https://pypi.org/p/sphinx_minoo_theme)
- [ ] [sphinx_ops_theme](https://pypi.org/p/sphinx_ops_theme)
- [ ] [sphinx_py3doc_enhanced_theme](https://pypi.org/p/sphinx_py3doc_enhanced_theme)
- [x] [sphinx_rtd_theme](https://pypi.org/p/sphinx_rtd_theme) -- #19
- [x] [sphinx_sizzle_theme](https://pypi.org/p/sphinx_sizzle_theme)
- [x] [sphinx_theme_pd](https://pypi.org/p/sphinx_theme_pd) -- #55
- [x] [sphinx-better-theme](https://pypi.org/p/sphinx-better-theme) -- #56
- [x] [sphinx-boost](https://pypi.org/p/sphinx-boost) -- #60
- [x] [sphinx-bootstrap-theme](https://pypi.org/p/sphinx-bootstrap-theme) -- #60
- [x] [sphinx-bulma-theme](https://pypi.org/p/sphinx-bulma-theme) -- #60
- [x] [sphinx-catalystcloud-theme](https://pypi.org/p/sphinx-catalystcloud-theme) -- #60
- [x] [sphinx-corlab-theme](https://pypi.org/p/sphinx-corlab-theme) -- #60
- [x] [sphinx-fossasia-theme](https://pypi.org/p/sphinx-fossasia-theme) -- #60
- [x] [sphinx-foundation-theme](https://pypi.org/p/sphinx-foundation-theme) -- #60
- [x] [sphinx-glpi-theme](https://pypi.org/p/sphinx-glpi-theme)
- [ ] [sphinx-guillotina-theme](https://pypi.org/p/sphinx-guillotina-theme)
- [x] [sphinx-ioam-theme](https://pypi.org/p/sphinx-ioam-theme) -- #60
- [x] [sphinx-kr-theme](https://pypi.org/p/sphinx-kr-theme) -- #60
- [x] [sphinx-material](https://pypi.org/p/sphinx-material) -- #60
- [x] [sphinx-modern-theme](https://pypi.org/p/sphinx-modern-theme) -- #60
- [x] [sphinx-nameko-mw-theme](https://pypi.org/p/sphinx-nameko-mw-theme)
- [x] [sphinx-nameko-theme](https://pypi.org/p/sphinx-nameko-theme)
- [x] [sphinx-nervproject-theme](https://pypi.org/p/sphinx-nervproject-theme) -- #60
- [x] [sphinx-opnfv-theme](https://pypi.org/p/sphinx-opnfv-theme) -- #60
- [x] [sphinx-pdj-theme](https://pypi.org/p/sphinx-pdj-theme) -- #60
- [x] [sphinx-press-theme](https://pypi.org/p/sphinx-press-theme) -- #19
- [x] [sphinx-redactor-theme](https://pypi.org/p/sphinx-redactor-theme) -- #60
- [x] [sphinx-rigado-theme](https://pypi.org/p/sphinx-rigado-theme) -- #60
- [x] [sphinx-susiai-theme](https://pypi.org/p/sphinx-susiai-theme) -- #59
- [ ] ~[sphinx-theme](https://pypi.org/p/sphinx-theme)~
- [x] [sphinx-typlog-theme](https://pypi.org/p/sphinx-typlog-theme) -- #27 
- [x] [sphinx-typo3-theme](https://pypi.org/p/sphinx-typo3-theme) -- #19
- [x] [sphinx-ustack-theme](https://pypi.org/p/sphinx-ustack-theme) -- #52
- [x] [sphinxbootstrap4theme](https://pypi.org/p/sphinxbootstrap4theme)
- [x] [sphinxcontrib-rextheme](https://pypi.org/p/sphinxcontrib-rextheme) -- borked. #57
- [x] [sphinxjp.themes.basicstrap](https://pypi.org/p/sphinxjp.themes.basicstrap)
- [ ] [sphinxjp.themes.dotted](https://pypi.org/p/sphinxjp.themes.dotted)
- [x] [sphinxjp.themes.gopher](https://pypi.org/p/sphinxjp.themes.gopher) -- borked.
- [ ] [sphinxjp.themes.htmlslide](https://pypi.org/p/sphinxjp.themes.htmlslide)
- [ ] [sphinxjp.themes.revealjs](https://pypi.org/p/sphinxjp.themes.revealjs)
- [ ] [sphinxjp.themes.s6](https://pypi.org/p/sphinxjp.themes.s6)
- [ ] [sphinxjp.themes.sphinxjp](https://pypi.org/p/sphinxjp.themes.sphinxjp)
- [x] [sphinxjp.themes.trstyle](https://pypi.org/p/sphinxjp.themes.trstyle) -- #59
- [x] [sponge-docs-theme](https://pypi.org/p/sponge-docs-theme) -- instructions unclear
- [x] [stanford-theme](https://pypi.org/p/stanford-theme)
- [ ] [stsci-rtd-theme](https://pypi.org/p/stsci-rtd-theme)
- [x] [sunpy-sphinx-theme](https://pypi.org/p/sunpy-sphinx-theme) -- #59
- [x] [t3sphinxthemertd](https://pypi.org/p/t3sphinxthemertd) -- replaced by sphinx-typo3-theme
- [x] [tibas.tt](https://pypi.org/p/tibas.tt) -- #59
- [x] [topos-theme](https://pypi.org/p/topos-theme) -- #59
- [x] [wild_sphinx_theme](https://pypi.org/p/wild_sphinx_theme) -- #59
- [x] [yummy-sphinx-theme](https://pypi.org/p/yummy-sphinx-theme) -- #59
- [x] [zerovm-sphinx-theme](https://pypi.org/p/zerovm-sphinx-theme) -- #59

Additional:

- [x] [Furo](https://pypi.org/p/furo)
- [x] [sphinx-uedoc-theme](https://pypi.org/p/sphinx-uedoc-theme)

**Comment by mgeier** [comment]
`mkdocs-nature` seems to have been included in error.

This is just the built-in `nature` theme, see those two for comparison:

* http://sphinx-themes.org/html/mkdocs-nature/nature/index.html
* http://sphinx-themes.org/html/default/nature/index.html

**Comment by mgeier** [comment]
`renga` seems to have been renamed to `renku` (and the installation has been slightly modernized)

The URL https://github.com/SwissDataScienceCenter/renga-sphinx-theme redirects to "renku".

Both look the same (except for the search button):

* http://sphinx-themes.org/html/renga-sphinx-theme/renga/index.html
* http://sphinx-themes.org/html/renku-sphinx-theme/renku/index.html

So probably we don't need to list `renga`?
See #46.

**Comment by pradyunsg** [comment]
Hey -- thanks for all the PRs toward this @mgeier! Much apprecaited! ^>^

I'm wondering what the situation would be w.r.t. merge conflicts, but I guess there's only one way to find out. :P

**Comment by mgeier** [comment]
Merge conflicts will always happen when inserting two different things between the same two list items. That's natural.

I think this can only be avoided be making multiple changes at once in one PR or by creating PRs more slowly ...

**Comment by pradyunsg** [comment]
Yup yup. I don't think I know of any trivial way to resolve that concern w/o changing how we handle themes. :)

**Comment by pradyunsg** [comment]
@shirou Any chance we could add some indicator to point folks who reach sphinx-themes.org to the v2 website while this issue is being iterated upon?

**Comment by shirou** [comment]
I'm very grateful for your remarkable works. But, sorry, I don't want to switch or navigate to v2 until at least more than half themes can be found in v2.

**Comment by pradyunsg** [comment]
Oh for sure. I don't want us to take down the current website until we get most of the themes onto this one.

Let me rephrase what I was asking -- can we add a link to the header, on the current sphinx-themes.org website, to point users to the newer design? :)

**Comment by shirou** [comment]
Can I ask why you want to add navigation to the current web page?

**Comment by pradyunsg** [comment]
@mgeier Thanks for all that you're doing here! ^>^

Are there any themes that you're unable to get working? If so, I'd like to check those off, much like I have for `cloud_sptheme` and similar above.

---

@shirou I've proposed a PR to provide more emphasis on "sphinx-themes.org as the gallery of third-party themes" in Sphinx's documentation about third-party themes. Part of the pitch being that we have a newer design coming up.

Most websites usually have a small period where they mention "hey, checkout our new design" for a period before switching over, so it'd be nice to follow that model here too. I'm mostly thinking that we should surface the "there's a v2 in the works" information on sphinx-themes.org itself, so that it's clear that the v2 website is indeed intended as a future replacement. :)

**Comment by pradyunsg** [comment]
FWIW, we are at 46 themes listed on v2, with ~12 themes not being added due to the themes being no-longer-installable or "not for general use".

That's compared to 115 on the older site -- which also allowed duplication (multiple listings for a single theme).

**Comment by mgeier** [comment]
> Are there any themes that you're unable to get working?

No, not so far.

I've just skipped `psphinxtheme` and `pylons-sphinx-themes` because I didn't know how to handle the multiple variants of each theme.

> If so, I'd like to check those off, much like I have for `cloud_sptheme` and similar above.

The setup of `cloud` is simple, see #53. I've also used it there: https://github.com/spatialaudio/nbsphinx/compare/cloud-theme%5E...cloud-theme.

I didn't look for instructions for any of the themes I ported. I just used the previous `conf.py` from the legacy http://sphinx-themes.org/ site and transferred it.

I think some of the unmaintained themes should be filtered out (which is not my choice to make), but lack of documentation IMHO shouldn't be the reason.

**Comment by pradyunsg** [comment]
> lack of documentation IMHO shouldn't be the reason

I strongly think that it should.

I'm not sure that I like the idea of adding projects that are meant for reuse, but don't have documentation on how to actually use them,are not done well. Especially when it's the result of investing effort to write a theme for documentation (which shows that you're willing to invest effort into documentation related stuff).

**Comment by mgeier** [comment]
> I strongly think that it should.

Sure, that's totally up to you. I'm not the one curating this list.

It would just be a pity not to show the `cloud` theme, since it's one of the better themes and it has some really original features.

> None the less, we can get to dropping packages at a later date if we hear that they can't be used.

Yeah. I'm just porting a few themes, feel free to drop any of them at any time.

**Comment by pradyunsg** [comment]
Sure thing. Thanks for all that you're doing here. ^.^

**Comment by mgeier** [comment]
The `sponge-docs-theme` uses this in the old website:

```
html_theme = 'sphinx_rtd_theme'
```

Doesn't seem right, does it?
So this should be either updated or removed.

**Comment by mgeier** [comment]
`sphinx-theme` seems to appear twice; once as `stanford_theme` and once using `neo_rtd_theme`.
What's that all about?

**Comment by mgeier** [comment]
`sphinxjp.themes.gopher` is broken because of a typo in their `setup.py`: https://github.com/tell-k/sphinxjp.themes.gopher/blob/c7d051f30a7e15ef377eb45262c2759f19b4c9ea/setup.py#L76

**Comment by pradyunsg** [comment]
Thanks a ton for all that you've done here @mgeier. ^.^

---

@shirou I think we're ready to make the switch.

**Comment by mgeier** [comment]
Any news on this?

**Comment by pradyunsg** [comment]
Nudge @shirou. I think we can make the switch to v2 now.

**Comment by shirou** [comment]
I have updated DNS settings. And now, https://sphinx-themes.org/ is v2! Awesome work! Thank you so much!

**Comment by pradyunsg** [comment]
Hurray! I'm also gonna call this issue resolved since we've basically got most of the theme's on the v2 site!
